wikimedia/wikipedia-ios
# Image Recommendations - Hide feature when VoiceOver is on

**Phabricator:** https://phabricator.wikimedia.org/T360035

### Notes
* This feature hides the feature when VoiceOver is activated
* It's not being done upon NotificationCenter `UIAccessibilityVoiceOverStatusDidChangeNotification` assuming a user would have it on when launching the app, avoiding an explore feed refresh and the operation of removing and inserting the card
* So I'm checking if VoiceOver is on when doing all the checks to display the card 

### Test Steps
1. Run a fresh install of the app with VoiceOver already on
2. Make sure the new feature modal, explore feed card, and explore feed setting option for VoiceOver are not showing
3. Turn VoiceOver off, run another fresh install - make sure all the above items appear normally
